setup.py

```diff
@@ -179,6 +179,9 @@ def run(self):
     test_requirements=[
         'pytest',
         'coverage',
+        'types-setuptools',
+        'types-atomicwrites',
+        'types-python-dateutil'
```

Comment:
NB: adding these here didn't resolve the build error, that was fixed with the change to tox.ini below. So it's possible this is completely wrong.